core/src/test/scala/filodb.core/memstore/ratelimit/RocksDbCardinalityStoreMemoryCapSpec.scala

```diff
@@ -48,7 +48,7 @@ class RocksDbCardinalityStoreMemoryCapSpec  extends AnyFunSpec with Matchers {
     val timePerIncrementMicroSecs = (end-start) / numTimeSeries / 1000
     println(s"Was able to increment $numTimeSeries time series, $timePerIncrementMicroSecs" +
       s"us each increment total of ${totalTimeSecs}s")
-    timePerIncrementMicroSecs should be < 200L
+    timePerIncrementMicroSecs should be < 300L
```

Comment:
Not a big fan of putting assertions on runtimes in Spec these may lead to flaky tests based on the different platform.